src/pages/AdminDashboard/CreateRoomForm.tsx

```diff
@@ -206,7 +273,7 @@ const CreateRoomForm = ({ onFormSubmit }) => {
                         </Typography>
                         <Field
                             size="small"
-                            name="participants"
+                            name="destinations"
```

Comment:
Is Participants still displayed? It seems to be a better if for a user point of view rather than "destinations"

Reply:
I think we should use `attendees` (as google/outlook) instead of destinations in both back and frontend, as a domain name. I changed that to `destinations` to match the API response for the moment although